src/server/gc_worker/gc_worker.rs

```diff
@@ -597,6 +604,48 @@ where
         Ok(())
     }
 
+    fn update_cf_stats_map(&mut self, key_mode: GcKeyMode, cf_static: CfStatistics, cf: &str) {
```

Comment:
How about just add a method to return `Statistics` object for a given `key_mode` ? The caller knowns what to do next (e.g, invoke `mut_cf_statistics`, or `get_by_enum`).

Reply:
I found if we want to insert a key-value pair in map, 
we can use ```map.entry(key).or_insert(init_value)```

It can do that：
1. Insert a new key-value pair ,when the key is not exist.
2. Don‘t insert the  'init_value' , when the key is in the map.

so I don't need to add a method to do this .
Just use ```map.entry(key).or_insert(init_value)```